src/tools/mesh_tester.cpp

```diff
@@ -79,15 +114,44 @@ struct Input
   void fixOutfilePath();
 };
 
+const std::set<RuntimePolicy> Input::s_validPolicies({
```

Comment:
I thought I commented on this before. 

Is the whole purpose of storing the policies in a set here to validate the command line input?

Couldn't you get by with just the enum:
```
enum RuntimePolicy { 
    seq = 0,
#ifdef AXOM_USE_RAJA 
    raja_seq = 1, 
#ifdef AXOM_USE_OPENMP
    raja_omp = 2,
#endif
#ifdef AXOM_USE_CUDA
    raja_cuda = 3,
#endif
#endif /* AXOM_USE_RAJA */
    NUM_RUNTIME_POLICIES
};
```
Then you can validate the input based on the `NUM_RUNTIME_POLICIES` ? Would that be sufficient?

Reply:
No -- it actually validates the options at runtime. 

E.g. option '3' is only valid when the code is configured with ``raja`` and ``cuda`` and if you try to use that option in other builds, it throws an error with a nice message about why that option wasn't valid.

Reply:
You would be able to do runtime validation with just the `enum` by just adding the `NUM_RUNTIME_POLICIES` entry right?

Reply:
The runtime validation is handled for us by ``cli11`` via the ``std::set`` and automatically produces good error messages for us with very little code on our side.

What would be a benefit of having a count of the number of runtime policies?

Reply:
oh, OK, nevermind, I didn't realize that the `cli11` required an `std::set`. 

I tend to use the count to check manually if the supplied argument is within range. Doesn't `cli11` support a valid range without having to specify a `set`?

Reply:
@gzagaris -- ``cli11`` would support a range-based validation, but the set-based one is better in this case, since it provides automatic error messages for invalid options that are within the range.

Reply:
Thanks @kennyweiss good to know....